### PR TITLE
Update ECDH curves recommended by Mozilla, now that we are on stretch

### PR DESCRIPTION
**Pull request by liberodark**
Better security is option is not recommended.

## The problem

...

## Solution

...

## PR Status

...

## How to test

...

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

**Comment by frju365** [comment]
Can you add a description of source and perhaps of example of online conf which had removed it ?

For the conf I based my conf on this [conf](https://github.com/stylersnico/nginx-secure-config/blob/master/nginx.conf) but it's doesn't remove this part.

**Comment by liberodark** [comment]
Yes based on this work https://doc.probetech.ovh/services_sec/services_nginx/#our_ssl_certificate_a_approved_with_full_compliance

File: data/templates/nginx/server.tpl.conf [hunk 1]
```diff
@@ -37,7 +37,7 @@ server {
     # ssl_ecdh_curve secp521r1:secp384r1:prime256v1;
```
Comment (alexAubin) [inline data/templates/nginx/server.tpl.conf]: Note the comment here, which apparently was meant to be used once on Stretch...

**Comment by frju365** [comment]
But, we can perhaps enable stretch compatibility (the curves which were in comment)

**Comment by alexAubin** [comment]
>But, we can perhaps enable stretch compatibility (the curves which were in comment)

Yes :+1:

**Comment by alexAubin** [comment]
So as discussed, we shall instead use the curves recommended by Mozilla, now that they are available on Stretch.

I'm yolomerging this as it was already agreed in previous PR (we just fell back to curves available on jessie while we were waiting for stretch)

**Pull request merged**